frontend/packages/console-shared/src/components/markdown-extensions/const.ts

```diff
@@ -0,0 +1,3 @@
+export const MARKDOWN_COPY_BUTTON_ID = 'data-copy-for';
+export const MARKDOWN_EXECUTE_BUTTON_ID = 'data-execute-for';
+export const MARKDOWN_SNIPPET_ID = 'data-snippet-id';
```

Comment:
These consts could move to their respective extension file.

Reply:
These consts are being shared by all of the extensions. I think it's good to keep them in a separate file.

Reply:
Sorry when i made the comment I didn't realize they were shared. I thought I had deleted this comment afterwards :) I guess I didn't!